code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -268,7 +307,7 @@
 		audible_message("<span class='danger'>[src] buzzes oddly!</span>")
 		flick("medibot_spark", src)
 		if(user)
-			oldpatient = user
+			previous_patient = user.UID()
```

Comment:
Why is this section of code using UIDs when the ignore list didn't?

Reply:
> Why is this section of code using UIDs when the ignore list didn't?

Ignore list is hotter than this. We don't need to bother with proper book-keeping here.